Themes/_fallback/metrics.ini

```diff
@@ -3089,6 +3089,7 @@ Line17="conf,SoundVolume"
 Line18="conf,EnableAttackSounds"
 Line19="conf,EnableMineHitSound"
 Line20="conf,VisualDelaySeconds"
+Line21="lua,RateModPreservesPitch()"

```

Comment:
I think it would be better to replace that with "lua,RateModPreservesPitch" and an entry in ScreenOptionsMasterPrefs.cpp. No need to reimplement the functionality in lua.

Reply:
Done. PTAL

Reply:
looks good now!